Cargo.toml

```diff
@@ -16,36 +22,87 @@ keywords = [
     "mozilla",
     "w3c",
     "webdriver",
-  ]
+]
 license = "MPL-2.0"
 repository = "https://hg.mozilla.org/mozilla-central/file/tip/testing/geckodriver"
 
 [dependencies]
-base64 = "0.13"
+anyhow = "1"
+base64 = "0.21"
 chrono = "0.4.6"
-clap = { version = "~3.1", default-features = false, features = ["cargo", "std", "suggestions", "wrap_help"] }
 hyper = "0.14"
 lazy_static = "1.0"
-log = { version = "0.4", features = ["std"] }
-marionette = "0.4.0"
-mozdevice = "0.5.1"
-mozprofile = "0.9.1"
-mozrunner = "0.15.1"
-mozversion = "0.5.1"
-regex = { version="1.0", default-features = false, features = ["perf", "std"] }
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 serde_yaml = "0.8"
 tempfile = "3"
+thiserror = "1"
 unicode-segmentation = "1.9"
-url = "2.0"
-uuid = { version = "1.0", features = ["v4"] }
-webdriver = "0.48.0"
-zip = { version = "0.6", default-features = false, features = ["deflate"] }
+url = "2.4"
+
+[dependencies.clap]
```

Comment:
These changes have been made by the `tomli_w` package. I'll revert those now by manually removing the `path` field.